core/test/TokenizedDerivative.js

```diff
@@ -2665,5 +2669,74 @@ contract("TokenizedDerivative", function(accounts) {
         )
       );
     });
+
+    it(annotateTitle("Oracle Fees"), async function() {
+      // A new TokenizedDerivative must be deployed before the start of each test case.
+      await deployNewTokenizedDerivative();
+
+      // Due to other test cases, the store could already have some balance in it.
+      let storeBalance = web3.utils.toBN(await getMarginBalance(deployedStore.address));
+      let totalOracleFeesPaid = web3.utils.toBN(web3.utils.toWei("0", "ether"));
+
+      // Deposit money into the short.
+      const initialBalance = web3.utils.toWei("1", "ether");
+      await derivativeContract.deposit(initialBalance, await getMarginParams(initialBalance));
+      shortBalance = (await derivativeContract.derivativeStorage()).shortBalance;
```

Comment:
nit: shouldn't this be `let shortBalance` (it doesn't look like it's declared above)?

Reply:
Yep, done.